docs/develop.adoc

```diff
@@ -74,36 +74,33 @@ history of `master`, depending on how they are merged.
   Merging commits like "Fix build" or "Implement comments from code review"
   should be avoided.
 
-== Continuous Integration (Travis CI)
+== Continuous Integration (Github Actions)
 
-Travis CI is used for continuously testing new commits and pull
-requests.
+Github actions are used for continuously testing new commits and pull requests.
+Those include testing for different operating systems, linting via clang-format and shellcheck,
+and code coverage and quality checks via `Codecov` and `LGTM.io`.
 
-We use the sudo-less beta Ubuntu Trusty containers, which do not permit
-root access.
-
-See the file `.travis.yml` and the scripts in `ci/` for the most
-up-to-date details.
+For Github workflows sources see `.github/workflows/` folder and scripts from the `ci/` folder.
+Also there is a Cirrus CI runner, configuration for which is stored in `.cirrus.yml`.
 
 === Reproducing Locally
 
-Sometimes tests fail in Travis CI and you will want to reproduce them
-locally for easier troubleshooting.
-
-We can use a container for this, like so:
+If tests fail in CI, you may attempt to reproduce those locally via `ctest` command:
 
 [source,console]
 --
-./travis.sh
+`ctest -j4 -V -R rnp_tests`
```

Comment:
Could remove the surrounding backticks.

Reply:
Thanks, will fix (as well as one below).